luarules/gadgets/unit_wanted_speed.lua

```diff
@@ -14,7 +14,7 @@ function gadget:GetInfo()
 		date      = "11 November 2018",
 		license   = "GNU GPL, v2 or later",
 		layer     = -9999999999, -- Before every state toggle gadget.
```

Comment:
Max (well in this case min) layer should be `-1000000` (one million), that is the maximum absolute value supported.